scripts/build-storybooks.js

```diff
@@ -22,15 +20,19 @@ const logger = console;
 
 const exec = async (command, args = [], options = {}) =>
   new Promise((resolve, reject) => {
-    const child = spawn(command, args, { ...options, stdio: 'inherit' });
-
-    child.on('close', code => {
-      if (code) {
-        reject();
-      } else {
-        resolve();
-      }
-    });
+    const child = spawn(command, args, { ...options, stdio: 'inherit', shell: true });
+
+    child
+      .on('close', code => {
+        if (code) {
+          reject();
+        } else {
+          resolve();
+        }
+      })
+      .on('error', e => {
+        logger.error(e);
```

Comment:
should it reject?

Reply:
Haven't changed the logic of this file
I added `shell: true` (windows fix) and an error output that was missing

It rejects `if (code > 0)` - maybe `code` should be renamed to `exitCode` ?

Reply:
yeah I should reject I think.